lib/local-storage-adapter.js

```diff
@@ -27,17 +34,34 @@ function createLocalStorageAdapter (queueName) {
     works: works,
     reset: reset,
     remove: remove,
-    type: 'localStorage'
+    type: 'localStorage',
+    flush: flush
   }
 
   return adapter
 
+  function flush () {
+    if (!storageInSync) {
+      adapter.save(queueKey, json.stringify(queueCache))
+      storageInSync = true
+      cacheInSync = false
```

Comment:
`cacheInSync` is set to false because we want to force a read from localStorage on the next get so that if any other queuing updated localStorage the extra items get picked up.